hello-world.js

```diff
@@ -1,19 +1,24 @@
-import Component from "can-component";
-import DefineMap from "can-define/map/map";
-import stache from "can-stache";
+import { Component, DefineMap } from "can";
 
-const HelloWorldVM = DefineMap.extend({
-  visible: { default: false },
-  message: { default: "Hello There!" }
+const ViewModel = DefineMap.extend({
+  visible: {
+    default: false
+  },
+  message: {
+    default: "Hello There!"
+  }
 });
 
 Component.extend({
   tag: "hello-world",
-  view: stache("{{#if visible}}{{message}}{{else}}Click me{{/if}}"),
-  ViewModel: HelloWorldVM,
-  events: {
-    click: function() {
-      this.viewModel.visible = !this.viewModel.visible;
-    }
-  }
+  view: `
+  <div on:click="this.visible = not(this.visible)">
+    {{# if(visible) }}
+      {{message}}
+    {{ else }}
+      Click me
+    {{/ if }}
+  </div>
+  `,
+  ViewModel
```

Comment:
We should probably put the ViewModel inline here.  But not super important.

Reply:
I thought about it, it was separated previously. I wasn't sure if it was useful to show it like that. I can change it though, no problem.

Reply:
If it is not inlined, I would suggest moving the `ViewModel` line up higher in the config. When looking back on code, it is very easy to miss a single line that's placed after a large block.